src/Tribe/Main.php

```diff
@@ -155,6 +156,16 @@ public function context_class() {
 		return $this->plugin_context_class;
 	}
 
+	/**
+	 * Initializes all libraries used/required by our singletons.
+
+	 *
+	 * @since TBD
+	 */
+	public function init_early_libraries(): void {
```

Comment:
```suggestion
	public function init_early_libraries() {
```
Check for others PHP 5.6 incompatibilities.